### PR TITLE
SWT-37 Add featured events to home page

### PR DESCRIPTION
**Pull request by aralatpulat**

### CODE REVIEW

**Comment by aralatpulat** [comment]
#37

File: Resources/Views/eventCard.leaf [hunk 1]
```diff
@@ -1,5 +1,12 @@
 <a href="/event/#(event.shortname)">
-    <div class="conference-cell-no-image">
+    <div class="event-cell">
         <p>#(event.fullname)</p>
+        #if(event.startDate != nil || event.endDate != nil) {
```
Comment (ahmetws) [inline Resources/Views/eventCard.leaf]: if you write if inside `<span class="date">`, we can omit the else clause.

File: Resources/Views/eventCard.leaf [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+            <span class="date">
+                #date(event.startDate, "dd MMM yyyy") - #date(event.endDate, "dd MMM yyyy")
+            </span>
+        } else {
+            <span class="date"></span>
```
Comment (ahmetws) [inline Resources/Views/eventCard.leaf]: we don't use the leading spaces after `if` and `else` in the leaf files for now. It is better to have the same style in all codebase if possible 👍

**Review by ahmetws — CHANGES_REQUESTED** [review]

File: Resources/Views/eventCard.leaf (hunk 1 above)
Comment (aralatpulat) [reply Resources/Views/eventCard.leaf]: good point. fixed.

File: Resources/Views/eventCard.leaf (hunk 2 above)
Comment (aralatpulat) [reply Resources/Views/eventCard.leaf]: fixed.

**Review by ahmetws — APPROVED** [review]

**Pull request merged**